src/api/external-window/external-window.ts

```diff
@@ -372,18 +376,19 @@ export class ExternalWindow extends EmitterBase<ExternalWindowEvents> {
      * Stops the taskbar icon from flashing.
      * @return {Promise.<void>}
      * @experimental
-     * @tutorial Window.stopFlashing
+     * @tutorial ExternalWindow.stopFlashing
      */
     public async stopFlashing(): Promise<void> {
         await this.wire.sendAction('stop-external-window-flashing', this.identity);
     }
 
     /**
+     * NOTE: Group methods on `ExternalWindow`s are currently unstable. This functionality will be addressed in a future update.
```

Comment:
UpdateOptions is not group method. Why need this note?

Reply:
Good catch. Thanks. Overzealous copy-pasting. That said, this method is actually not useful at the moment for two reasons:
1 - It doesn't actually work.
2 - The current set of ExternalWindow options are things we don't actually control without the injection dll, which we decided not to ship. (opacity, showTaskbarIcon, etc)

I'll remove this line now, but not completely sure what the best move is re: ExternalWindow options in general. Do you think we should remove the `update` method from the API?

I realize this is probably out of scope for this PR. Making a note to follow up on this tomorrow.

Reply:
probably add @ignore comment in jsdoc for this method if it's not working now.